app/templates/components/wds-global-navigation/link-button.hbs

```diff
@@ -0,0 +1,3 @@
+{{#wds-button class='wds-global-navigation__start-a-wiki' secondary=true}}
```

Comment:
Does this button work at all? I think we should add href=model.href to this button